test/Seq.App.EmailPlus.Tests/EmailReactorTests.cs

```diff
@@ -110,7 +110,7 @@ public void TrimStringHelper3Args()
 
 
         [Fact]
-        public void ToAddressesAreTemplated()
+        public async void ToAddressesAreTemplated()
```

Comment:
Should return `async Task` here, or xUnit can't track completion

Reply:
Now returning async Task.